_stbt/core.py

```diff
@@ -903,7 +902,7 @@ def _match_all(self, image, frame, match_parameters, region):
 
         grabbed_from_live = (frame is None)
         if grabbed_from_live:
-            frame = self._display.pull_frame()
+            frame = self._display.get_frame()
```

Comment:
Yay I was always confused about the proliferation of `get_frame` / `_get_sample` / `pull_frame`. Now we're rid of the latter 2.